soc/bouffalolab/soc.yml

```diff
@@ -0,0 +1,12 @@
+family:
+- name: bouffalolab_bflb
```

Comment:
Let's add a vendor here

Reply:
Could you provide the change you had applied and the build errors you got?

Reply:
I took a look on the schema and I found my error. The vendor should be after all entries at same level of family. Now it works!